Support for dark theme

filips123 opened the pull request:
Add support for the dark theme in ZeroMail.

Also, @HelloZeroNet, can you please raise my ZeroTalk limit for `filips@zeroid.bit`? I'm currently using `48.3k/50k`.

![screenshot](https://user-images.githubusercontent.com/16626308/60810527-a4de3f00-a18d-11e9-904e-9e593578b0f7.png)

HelloZeroNet commented:
Thanks, you don't need to add vendor-prefixes to the css, as the merger will do it for you automatically: 
https://github.com/HelloZeroNet/ZeroMail/pull/41/files#diff-fd9a87e290cf6a5ed1e7758b6164e72fR303

I already increased the limit earlier, but maybe forgot to publish it, so I did it again.

filips123 commented:
> Thanks, you don't need to add vendor-prefixes to the css, as the merger will do it for you automatically

Ok, I removed them from original CSS and removed duplicated from merged CSS.

> I already increased the limit earlier, but maybe forgot to publish it, so I did it again.

Great, thank you!

filips123 commented:
@HelloZeroNet Can you merge this?
